src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -783,8 +783,8 @@ Additional information: {1}</value>
   <data name="Manage_specifications" xml:space="preserve">
     <value>Manage specifications</value>
   </data>
-  <data name="Manage_styles" xml:space="preserve">
-    <value>Manage styles</value>
+  <data name="Manage_naming_styles" xml:space="preserve">
+    <value>Manage naming styles</value>
```

Comment:
why are we changing the value ?

Reply:
"Manage naming styles" was being displayed in the dialog box because that is the string that was hard-coded in `public string DialogTitle => "Manage Naming Styles"`.  However, the resource file had the localized string as "Manage styles".  I asked David which one we should go with and he said we should keep the one that was previously displayed.

Reply:
ah , got it ...the title and the bug description said otherwise so was confused.